bin/ddg2logo

```diff
@@ -179,14 +185,16 @@ parser = argparse.ArgumentParser(
 description=description, epilog=epilog, formatter_class=argparse.RawTextHelpFormatter)
 optional = parser._action_groups.pop()
 required = parser.add_argument_group('required arguments')
+group = parser.add_mutually_exclusive_group(required=True)
```

Comment:
can you call this variable something more explicit, like `thresholds`?

Reply:
Changed the name to `thresholds`